src/ctia/task/migration/migrate_es_stores.clj

```diff
@@ -291,17 +294,23 @@
                         index))))))
   true)
 
-(s/defn ^:always-validate run-migration
-  [{:keys [prefix
-           restart?]
-    :as params} :- MigrationParams]
-  (when-not restart?
-    (assert prefix "Please provide an indexname prefix for target store creation"))
+(s/defn get-migration-params :- MigrationParams
```

Comment:
The naming seems a bit off, this is not a "getter" so to call it.

I'd move the `(p/get-in-global-properties [:ctia :migration])` away, to `run-migration`, making this defn a pure function.

This pure function would be easier to name, and also would not need a `with-redefs` in the tests.